server/models/User.js

```diff
@@ -0,0 +1,393 @@
+        minlength: 3
+    },
+    startDate: {
+        type: Date,
+        trim: true,
+        minlength: 3
+    },
+    endDate: {
+        type: Date,
+        trim: true,
+        minlength: 3
+    },
+    description: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+})
+
+const experienceSchema = new Schema({
+    company: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    title: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    location: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    startDate: {
+        type: Date,
+        trim: true,
+        minlength: 3
+    },
+    endDate: {
+        type: Date,
+        trim: true,
+        minlength: 3
+    },
+    description: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+})
+
+const certificationSchema = new Schema({
+    title: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    issuingOrganization: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    issueDate: {
+        type: Date,
+        trim: true,
+        minlength: 3
+    },
+    expirationDate: {
+        type: Date,
+        trim: true,
+        minlength: 3
+    },
+    credentialID: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    credentialURL: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+})
+
+const awardSchema = new Schema({
+    title: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    issuingOrganization: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    issueDate: {
+        type: Date,
+        trim: true,
+        minlength: 3
+    },
+    description: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+})
+
+const publicationSchema = new Schema({
+    title: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    publisher: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    publicationDate: {
+        type: Date,
+        trim: true,
+        minlength: 3
+    },
+    description: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+})
+
+const projectSchema = new Schema({
+    title: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    link: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    description: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+})
+
+const languageSchema = new Schema({
+    name: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+    proficiency: {
+        type: String,
+        trim: true,
+        minlength: 3
+    },
+})
+
+const userSchema = new Schema({
+    auth: {
+        email: {
+            type: String,
+            required: true,
```

Comment:
Customize validation error messages to provide more meaningful feedback. This can be especially helpful during development and debugging
` required: [true, 'Email is required.'],`

Reply:
change for all like this